docs/reST/ref/draw.rst

```diff
@@ -105,16 +105,16 @@ object around the draw calls (see :func:`pygame.Surface.lock` and
 .. function:: circle
 
    | :sl:`draw a circle`
-   | :sg:`circle(surface=Surface, color=Color, center=(x, y), radius=radius) -> Rect`
-   | :sg:`circle(surface=Surface, color=Color, center=(x, y), radius=radius, width=0) -> Rect`
+   | :sg:`circle(surface, color, center, radius) -> Rect`
+   | :sg:`circle(surface, color, center, radius, width=0) -> Rect`
 
    Draws an circle on the given surface.
 
    :param Surface surface: surface to draw on
    :param color: color to draw with, the alpha value is optional if using a
       tuple ``(RGB[A])``
    :type color: Color or int or tuple(int, int, int, [int])
-   :param center: center point of the circle
+   :param center: center point of the circle, (x, y)
    :type center: tuple(int, int) or list(int, int)
```

Comment:
center now takes floats as well as int. Also Vector2 works. Not sure how to express all this exactly.

Reply:
Thanks. I opened #1040 to track this.